.github/CONTRIBUTING.md

```diff
@@ -81,6 +99,20 @@ $ make html
 
 The built documentation can then be found in `docs/_build/html/`.
 
+To file a pull request, create a new branch on top of the upstream repository:
+
+```console
+$ git checkout -b my_topical_branch upstream/main
```

Comment:
I think I prefer a `git fetch --all; git checkout` over this monstrosity.

Traditionally it hasn't been a problem if someone didn't have an up-to-date repo since there's the sweet update branch button. I'd really like to keep this as simple as possible.

Reply:
Sure, this example takes things to extremes. Though, it's still a good demonstration because it's accurate.